docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-flag-card-group/__tests__/cgdp-flag-card-group.dom.ts

```diff
@@ -0,0 +1,71 @@
+export const cgdpFlagCardGroupDom = () => {
+	const div = document.createElement('div');
+
+	// language=HTML
+	div.innerHTML = `
+		<div class="usa-section" data-eddl-landing-row="">
+      <h2 class="nci-heading-h3 padding-bottom-2">
+				The Optional Flag Card Group Heading
+			</h2>
+			<ul class="nci-card-group cgdp-flag-card-group">
+				<li class="nci-card cgdp-flag-card">
```

Comment:
can ya make sure the "good" html is good? looks like there's some left over nci-card stuff here